src/test/resources/samples/invalid/eli_courtwright.ics

```diff
@@ -1,20 +1,20 @@
-BEGIN:VCALENDAR
-VERSION:2.0
-CALSCALE:GREGORIAN
-BEGIN:VEVENT
-DURATION:PT1H
-LOCATION:515 East Capitol Street SE\, Washington\, DC 20003
-DTSTAMP:20040319T053019Z
-UID:41E603E9-5C76-11D9-B268-000D93514BB4
-SEQUENCE:1
-DTSTART;TZID=US/Eastern:20040307T184500
-SUMMARY:Potluck and Dialogue at William Penn House: David Etheridge of F
- riends Meeting of Washington on Humor in the Bible
-DESCRIPTION:The focus of the discussion will be on Biblical humor as int
- ended by the author.\n\nOur monthly potluck is the first Sunday of each 
- month at 6:30 PM. This is an opportunity for fellowship among Quakers a
- nd attenders. Bring a dish to share\; family members and friends are al
- ways welcome. Note: William Penn House is an alcohol and tobacco free f
- acility.
-END:VEVENT
-END:VCALENDAR
+BEGIN:VCALENDAR
```

Comment:
Is the change for the ICS test files relating to line endings? Not sure if this was intentional or not..

Reply:
I'd thought it was only encoding. As soon as I pulled the repo a bunch of the ics files had shown changes and I was unable to revert or stash them them as it just kept happening automatically. I thought that perhaps Intellij was changing them to UTF-8 so I closed the IDE and tried reverting them in terminal but still the same. I ended up just committing them as I couldn't work around it.

Reply:
It's weird because I am also using IntelliJ, but when I check out your fork it also applies changes to revert the line endings. 

Note that there is a `.gitattributes` file to force the `CRLF` line endings for *.ics files, but for some reason it doesn't seem to work for you I guess. If I may I might try to revert this in your fork to reduce the noise in the PR.